strawberryfields/decompositions.py

```diff
@@ -476,6 +476,190 @@ def rectangular_symmetric(V, tol=1e-11):
     return new_tlist, new_diags, None
 
 
+def BS_mat(theta):
+    r"""Beam splitter matrix as implemented in hardware
+    """
```

Comment:
Best to add more detail to this docstring for future developers :) e.g.
```suggestion
    r"""Beamsplitter matrix as implemented in hardware.
    
    Args:
        theta (float): Transmittivity angle :math:`\theta`. The transmission amplitude of
            the beamsplitter is :math:`t = \cos(\theta)`.
    """
```